contracts/0.8.25/vaults/dashboard/Dashboard.sol

```diff
@@ -194,24 +208,58 @@ contract Dashboard is NodeOperatorFee {
      * @dev Automatically funds the staking vault with ether
      */
     receive() external payable {
-        _fund(msg.value);
+        if (_shouldFundOnReceive()) _fund(msg.value);
     }
 
     /**
-     * @notice Transfers ownership of the staking vault to a new owner.
+     * @notice Sets the owner of the staking vault.
      * @param _newOwner Address of the new owner.
      */
-    function transferStakingVaultOwnership(address _newOwner) external {
-        _transferStakingVaultOwnership(_newOwner);
+    function setVaultOwner(address _newOwner) external {
+        _setVaultOwner(_newOwner);
     }
 
     /**
      * @notice Disconnects the staking vault from the vault hub.
+     * VaultHub stores data for calculating the node operator fee, so the fee is disbursed first.
      */
     function voluntaryDisconnect() external {
+        disburseNodeOperatorFee();
+
         _voluntaryDisconnect();
     }
 
+    /**
+     * @notice Accepts the ownership over the staking vault transferred from VaultHub on disconnect
+     * and immediately transfers it to a new pending owner. This new owner will have to accept the ownership
+     * on the staking vault contract.
+     * @param _newOwner The address to transfer the staking vault ownership to.
+     */
+    function abandonDashboard(address _newOwner) external {
+        address vaultAddress = address(_stakingVault());
+        if (VAULT_HUB.vaultConnection(vaultAddress).vaultIndex != 0) revert ConnectedToVaultHub();
+
+        _acceptOwnership();
+        _transferOwnership(_newOwner);
+    }
+
+    /**
+     * @notice Accepts the ownership over the staking vault and connects to VaultHub.
+     */
+    function acceptOwnershipAndConnectToVaultHub() external payable {
+        _acceptOwnership();
+        connectToVaultHub();
+    }
+
+    /**
+     * @notice Connects to VaultHub, transferring ownership to VaultHub.
+     */
+    function connectToVaultHub() public payable {
+        if (msg.value > 0) _stakingVault().fund{value: msg.value}();
```

Comment:
:thinking: could be fundable after global permission revamp merged in (allowing role admin to execute on behalf subroles)

Reply:
No, it's using the direct fund to vault, not a fund through VaultHub like in fundable